modules/core/shared/src/main/scala/lucuma/core/util/Display.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.util
+
+/**
+  * Typeclass for things that can be shown in a user interface.
+  * @group Typeclasses
+  */
+trait Display[A]     {
+  def shortName(a: A): String
+  def longName(a:  A): String = shortName(a)
+}
```

Comment:
Sorry, late to the party.

Should we add some syntax to permit `a.shortName` and `a.longName` in a future PR?  And an

```scala
object Display {
  def apply[A](implicit ev: Display[A]): Display[A] = ev
}
```

for convenient summons?

Reply:
Great idea

Reply:
Yes, good idea.

Reply:
Guess I was a little quick with the merge button. I'll create another PR with this.